test/module/evaluators.js

```diff
@@ -76,6 +76,56 @@ test('createSafeEvaluator', t => {
   Function.__proto__.constructor.restore();
 });
 
+test('createSafeEvaluatorWhichTakesEndowments - options.transforms', t => {
+  t.plan(4);
+
+  // Mimic repairFunctions.
+  // eslint-disable-next-line no-proto
+  sinon.stub(Function.__proto__, 'constructor').callsFake(() => {
```

Comment:
Just noting what we discussed verbally: I find this whole `sinon` mocking thing to be disturbingly weird, but I have not investigated. In any case, no need to delay this PR.